### PR TITLE
New events for activating / deactivating users

### PR DESCRIPTION
**Pull request by tomaszszymanski129**
I want to merge this change because it adds two new customer events: `account_activated` and `account_deactivated`.
image

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [x] The changes are tested
* [x] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-7583 Create an event for activating and deactivating users](https://app.clickup.com/t/2549495/SALEOR-7583)

File: saleor/account/events.py [hunk 1]
```diff
@@ -14,6 +14,32 @@ def customer_account_created_event(*, user: User) -> Optional[CustomerEvent]:
...
+def customer_account_deactivated_event(
+    *, staff_user: UserType, app: AppType, account_id: int
+) -> Optional[CustomerEvent]:
+    if not user_is_valid(staff_user):
+        staff_user = None
+    return CustomerEvent.objects.create(
+        user=staff_user,
+        app=app,
+        type=CustomerEvents.ACCOUNT_DEACTIVATED,
+        parameters={"account_id": account_id},
```
Comment (korycins) [inline saleor/account/events.py:39]: Don't you also need to extend `CustomerEvent` type to provide field `accountID`?

Comment (korycins) [reply saleor/account/events.py:39]: If yes, just reminder that the `id` should be converted to graphql form

Comment (maarcingebala) [reply saleor/account/events.py:39]: Should it? I think for other types we store database ID. We should rather translate it in some resolver in the corresponding GraphQL type.

**Review by IKarbowiak — APPROVED** [review]

**Review by maarcingebala — APPROVED** [review]

**Comment by patrys** [comment]
@tomaszszymanski129 Why do we use `account_id` and not `user_id`? There is no Account type in the API and `account*` mutations all contain a `user` field.

**Pull request merged**